docs/NewFeatureProcess.md

```diff
@@ -12,73 +12,30 @@ product. The [Feature Tracking board](https://github.com/Microsoft/calculator/pr
 
-* **Problem Statement**: What problem are we trying to solve? Who’s the target audience? Is there a
-  customer need or pain point we need to remedy? Is there a business goal or metric we are trying
-  to improve? Do we have a hypothesis we want to prove or disprove?
-* **Evidence or User Insights**: Why should we do this? Potential sources of data: Feedback Hub,
-  other GitHub issues, other anecdotes from listening to customers in person or online, request
-  from another team, telemetry data, user research, market or competitive research
-* **Proposal**: How will the solution/feature help us solve the problem? How will it meet the
-  target audience’s needs? If there are business goals or metrics, how does this improve them?
-* **Goals**: What you want to accomplish with this feature. Typical examples include “User Can
-  *perform some task*”
-* **Non-Goals**: Things we are explicitly not doing or supporting or that are out of scope,
-  including reasons why.
-* **Low-Fidelity Concept**: Show as much of the experience as needed to explain the idea. This
-  can be as simple as a napkin drawing but can also be a code prototype, a PowerPoint walkthrough,
-  or a design comp.
-
-The low-fidelity concept should be kept simple at this stage and refined during the pre-production
-process.
-
-Feature pitches are submitted as issues on GitHub. We encourage discussion on open issues, and as
-discussion progresses we will edit the issue description to refine the idea.
+We review pitches regularly, and will approve or close issues based on whether they broadly aligns with the [Calculator roadmap](https://github.com/Microsoft/calculator/blob/master/docs/Roadmap.md). Approved pitches are moved into [pre-production](https://github.com/Microsoft/calculator/projects/1) on the feature tracking board.
 
 ## Step 2: Pre-production
-In the pre-production phase, we experiment with a variety of ways to address the goals described in
-the feature pitch. The output of this phase is a specification which demonstrates how the feature
-will work, supported by design renderings and code prototypes as needed. Sometimes we'll learn new
-things about a feature proposal during pre-production, and we'll edit or close the original pitch.
+For most features, the output of this phase is a specification which describes how the feature will work, supported by design renderings and code prototypes as needed. The original issue will continue to track the overall progress of the feature, but we will create and iterate on spec documentation in the [Calcualtor Spec repo](https://github.com/Microsoft/calculator-specs). Sometimes we'll learn new things about a feature proposal during pre-production, and we'll edit or close the original pitch.
```

Comment:
That seems to work for displayed content, but we should not apply this rule to comments in templates. Otherwise, you see this:

![image](https://user-images.githubusercontent.com/1414423/55652784-6e191700-57a1-11e9-9d7c-dc99244de59e.png)